tools/cldr-code/src/main/java/org/unicode/cldr/util/StandardCodes.java

```diff
@@ -67,6 +67,23 @@ public static CodeType from(String name) {
             }
             return CodeType.valueOf(name);
         }
+
+        public NameType toNameType() {
+            switch (this) {
+                case language:
+                    return NameType.LANGUAGE;
```

Comment:
This maps StandardCodes.CodeType to NameType. These enums are strangely overlapping, suggesting possible future integration